glossary.rst

```diff
@@ -141,3 +161,11 @@ Glossary
       Solver
          A resource that runs a problem. Some solvers interface to the :term:`QPU`;
          others leverage CPU and GPU resources.
+
+      Subgraph
+         See subgraph_.
+
+         .. _subgraph: https://en.wikipedia.org/wiki/Glossary_of_graph_theory_terms#subgraph
+
+      Working graph
+         In a D-Wave QPU, the set of qubits and couplers that are available for computation is known as the working graph. The yield of a working graph is typically less than the total number of qubits and couplers that are fabricated and physically present in the QPU.
```

Comment:
less than 100% of qubits and couplers that are fabricated

Reply:
That one is copy and pasted from you! :smile:  I'll change it